playwright/src/test/java/com/microsoft/playwright/TestGlobalFetch.java

```diff
@@ -370,6 +371,26 @@ void shouldThrowAnErrorWhenMaxRedirectsIsExceeded() {
     request.dispose();
   }
 
+  @Test
+  void shouldThrowAnErrorWhenMaxRedirectsOnContextIsExceeded() {
```

Comment:
Let's preserve the test name and port other tests from https://github.com/microsoft/playwright/pull/35160